packages/contentful--react-apps-toolkit/src/useCMAClient.ts

```diff
@@ -0,0 +1,36 @@
+import { createClient, PlainClientAPI } from "contentful-management";
+import { useEffect, useState } from "react";
+import { useSDK } from "./index";
+
+/**
+ * A react hook returning the plain client CMA.
+ *  - Depends on Apps SDK.
+ *  - Must be called in the SDKProvider component (./SDKProvider). If called outside the provider, will throw error. useSDK hook called on line 12 handles error.
```

Comment:
What about being deliberate, throw and exception and tell the user what's up? 
I think it's not ideal development experience if you get the error and you need to look through the source why. WDYT?

Reply:
The `useSDK` hook would throw an exception first if this hook is not wrapped in the `SDKProvider`. So it didn't seem to make sense to add an error here. But, now that you mention it, the error being thrown would be specific to the `useSDK` hook, which is confusing. So maybe we can make it more general. Like "Make sure this hook is being used inside of the SDKProvider".

Reply:
@shikaan If either hook is used outside of the provider, an error saying "Make sure this hook is used inside the SDKProvider" will be thrown. WDYT?